### ISSUE TITLE
make testing/rolling default branch

### ISSUE THREAD

**Issue by polandlover123** [spacefault/grade-server-nix#36]

**Comment by spacefault** [comment]
This may not work, because `system.autoUpgrade` looks for the main branch in the repository to pull from (see #40). Also, this would be unsafe because anything in the testing branch would instantly be deployed, which might cause issues for users.

**Comment by polandlover123** [comment]
no, i think you can set it to track "github:gradetools/minecraft-nix/rolling/unstable"

**Comment by spacefault** [comment]
oh, my apologies
however we should still consider having a production branch, because anything in production is usually marked for release
we could ditch the multi-branch thing completely, and just use other people's pull requests/forks *as* the testing branch. if we do this then it's *technically* rolling
